docker/Dockerfile

```diff
@@ -55,10 +55,15 @@ ARG VERSION="unknown"
 ARG GIT_COMMIT="unknown"
 RUN make gpu-operator
 
-FROM nvcr.io/nvidia/cuda:12.8.1-base-ubi9
+# Download must-gather dependency: `kubectl`
```

Comment:
I think we can avoid packaging `kubectl` binary too. This often have CVEs. For openshift, we can publish a minimal image into RH registry like [these](https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/support/gathering-cluster-data#gathering-data-specific-features_gathering-cluster-data). Which is used only during log collection using `oc adm must-gather --image <debug-image>`. For K8s users, they will use the script directly from the GH repo anyway.

Reply:
We currently still need it for the `upgrade-crd` and `cleanup-crd` helm hooks.

Reply:
For sure, Shiva. We do have that plan. So, It will be done in a follow-up PR.